src/details/ArborX_PairIndexRank.hpp

```diff
@@ -0,0 +1,28 @@
+/****************************************************************************
+ * Copyright (c) 2017-2023 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#ifndef ARBORX_PAIR_INDEX_RANK_HPP
+#define ARBORX_PAIR_INDEX_RANK_HPP
+
+#include <Kokkos_Macros.hpp>
+
```

Comment:
```suggestion
#include <Kokkos_Macros.hpp>

```

Reply:
I'm confused. The patch looks the same.

Reply:
I don't get it either...  Marking it as resolved.

Reply:
Oh no I meant you don't use it

Reply:
Editing the diff

Reply:
You will need to do locally.  I was trying to remove the extra blank line